arbnode/tx_pre_checker.go

```diff
@@ -35,6 +36,16 @@ const TxPreCheckerStrictnessAlwaysCompatible uint = 10
 const TxPreCheckerStrictnessLikelyCompatible uint = 20
 const TxPreCheckerStrictnessFullValidation uint = 30
 
+func MakeNonceError(sender common.Address, txNonce uint64, stateNonce uint64) error {
+	if txNonce < stateNonce {
+		return fmt.Errorf("%w: address %v, tx: %d state: %d", core.ErrNonceTooLow, sender, txNonce, stateNonce)
```

Comment:
Will more verbose error messages cause problems with dapps that expect geth style errors?

Reply:
This is consistent with our existing errors